app/javascript/pages/Orientation.vue

```diff
@@ -1,81 +1,125 @@
-<!--            <br><br>-->
-<!--            <span class=''><a href='/contributions'>View offers and resources [*]</a></span>-->
+      <div class='columns actions is-centered is-variable is-3'>
+        <div class='column is-one-third'>
+          <div class='box action'>
+            <h2 class='subtitle is-3'>Need some help?</h2>
+            <div class='action-buttons'>
+              <a href="/asks/new" class="button is-primary">Submit an Ask</a>
+            </div>
+            <div>
+              <a href="/contributions?ContributionType[Ask]=1">View asks</a>
+            </div>
           </div>
         </div>
-        <div>
-          <br><br>
-          <h2 class='subtitle is-3'>Want to help?</h2>
-          <div class='action-buttons'>
-            <a href="/offers/new" class="button is-primary">Submit an Offer</a>
-<!--            <a href="/contributions?contribution_type=ask" class="button is-primary is-outlined">View Asks [*]</a>-->
-            <br><br>
-            <span class=''><a href='/community_resources/new'>Submit a community resource</a></span>
-            <br>
-            <span class=''><a href='/announcements/new'>Submit an announcement</a></span>
-<!--            <br>-->
-<!--            <span class=''><a href='/donations/new'>Donate money or goods</a></span>-->
-<!--            <br>-->
-<!--            <span class=''><a href='/offers/new?page_title=Volunteer Request'>Volunteer [TBD]</a></span>-->
+        <div class='column is-one-third'>
+          <div class='box action'>
+            <h2 class='subtitle is-3'>Want to help?</h2>
+            <div class='action-buttons'>
+              <a href="/offers/new" class="button is-primary">Submit an Offer</a>
+            </div>
+            <div>
+              <a href="/contributions?ContributionType[Offer]=1">View offers</a>
+            </div>
           </div>
         </div>
-
-<!--        <div>-->
-<!--          <br><br>-->
-<!--          <h2 class='subtitle is-3'>Both, maybe?</h2>-->
-<!--          <div class='action-buttons'>-->
-<!--            <a href="/combined_form" class="button is-primary">Create a Profile [TBD]</a>-->
-<!--          </div>-->
-<!--        </div>-->
       </div>
-    </div>
+      <hr>
+      <div class='level'>
+        <h1 class='title level-item'>Mutual Aid</h1>
+      </div>
+      <hr>
+      <div class='columns'>
+        <div class='column'>
+          <AboutSection id='landing_page_text_what' title='What is Mutual Aid?' :description="landing_page_text_what"/>
+        </div>
+        <div class='column'>
+          <AboutSection id='landing_page_text_who' title='Who are we?' :description="landing_page_text_who"/>
+        </div>
+        <div class='column'>
+          <AboutSection id='landing_page_text_how' title="Where do you fit in?" :description="landing_page_text_how"/>
+        </div>
+      </div>
+      <hr>
```

Comment:
I really like how this section, with the `hr`s and the columns, comes out looking